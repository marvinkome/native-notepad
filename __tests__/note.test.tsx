/**
 * __tests__/note.test
 */

import * as React from 'react';
import renderer from 'react-test-renderer';
import Note from '../src/components/note';

it('Renders Notepage correctly', () => {
    const navigation = {
        navigate: jest.fn(),
        setParams: jest.fn(),
        getParam: jest.fn()
    };
    const note = renderer.create(<Note navigation={navigation} />);
    expect(note).toMatchSnapshot();
});
