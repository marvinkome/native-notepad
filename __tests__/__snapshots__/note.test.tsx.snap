// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Notepage correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "height": 1334,
      },
      Object {
        "backgroundColor": "#fff",
      },
    ]
  }
  virtual={undefined}
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        },
        Object {
          "paddingLeft": 10,
          "paddingTop": 20,
        },
      ]
    }
    viewIsInsideTabBar={false}
    virtual={undefined}
  >
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#000",
              "fontFamily": "Roboto",
              "fontSize": 16,
            },
            Object {
              "fontSize": 20,
              "fontWeight": "100",
            },
          ]
        }
        uppercase={false}
        virtual={undefined}
      />
    </View>
  </RCTScrollView>
</View>
`;
